terraform/modules/fhir/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "region" {
+  type        = string
+  description = "The region of the project and the related data sets"
+  default     = "us-west1"
+}
+
+variable "time_zone" {
+  type        = string
+  description = "The default timezone used by this dataset. Must be a either a valid IANA time zone name or empty, which defaults to UTC."
+  default     = "UTC"
+}
+
+variable "dataset_name" {
+  type        = string
+  description = "The resource name for the Dataset."
+  default     = "PHDI_DATASET"
+}
```

Comment:
According to [this doc](https://cloud.google.com/healthcare-api/docs/how-tos/datasets#healthcare-create-dataset-gcloud) `DATASET_ID` must be unique within a region. To ensure we can handle deploying to multiple GCP projects in the same region could we append a random number to `"PHDI_DATASET"` as we have done for storage bucket names in `storage/main.tf`?

Reply:
Agreed, I was thinking that the dataset name would be set by the client and we could even add a random number generator to add something to the end to make it even more unique.  Or just use the date it was created at the end of the name.